### PR TITLE
Project-happy-thoughts Petra Eriksson

### PR DESCRIPTION
**Pull request by petrasson**
Link to my deployed project:
https://project-happy-thought-petra.netlify.app/

My reflections on how this week's project turned out:
It went well at the beginning of the week but I was really struggling with the "like a tweet"... Spent a day trying to solve that. React is a bit confusing still but learning something new each week. At the end of the day, I am happy with the result.

### CODE REVIEW

File: code/src/components/TweetForm.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+const TweetForm = ({ newTweet, onNewTweetChange, handleFormSubmit }) => {
+  return (
+    <div className="form-container">
+      <form onSubmit={handleFormSubmit}>
+        <h1 className="title">Spread some love!</h1>
```
Comment (erilinus) [inline code/src/components/TweetForm.js:7]: Just nitty gritty design feedback... The heading is centered but not all the way in the middle of the desktop version. I would add `text-align: center;` to solve that, or remove the padding and make it left-aligned like the mobile version.

File: code/src/index.css [hunk 2]
```diff
@@ -1,13 +1,207 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
```
Comment (erilinus) [inline code/src/index.css:5]: Nice and clean!! Gotta start using this 🤩

File: code/src/index.css [hunk 3]
```diff
@@ -1,13 +1,207 @@
...
+  font-size: 12px;
+  color: var(--fifth);
+}
+.form {
+  width: auto;
+}
+
+.textarea-wrapper {
+  align-items: center;
+  justify-content: center;
```
Comment (erilinus) [inline code/src/index.css:85]: I'm not sure if `align-items` and `justify-content` work properly without `display: flexbox;` 🤔

File: code/src/index.css [hunk 4]
```diff
@@ -1,13 +1,207 @@
...
+  background-color: var(--fourth);
+}
+
+.heart-counter {
+  margin: 0;
+}
+
+.time-stamp {
+  justify-content: end;
+}
```
Comment (erilinus) [inline code/src/index.css:184]: Could be removed if cleanup. Alignment is done in `.like-time-box {justify-content: space-between;}`.

**Review by erilinus — COMMENTED** [review]
Great work Petra! 
I like that you've added your own styling to this project. Everything work as expected and it's on target with the requirements. I got great inspiration from reviewing your code!! 😃 

There are some small fixes I've added comments to and I noted that you've used flexbox properties without `display: flexbox;`. If you need it, [this website is great to understand flexbox more](https://css-tricks.com/snippets/css/a-guide-to-flexbox/).

File: code/src/App.js [hunk 5]
```diff
@@ -1,9 +1,78 @@
...
+import React, { useState, useEffect } from 'react';
+import TweetForm from 'components/TweetForm';
+import TweetList from 'components/TweetList';
 
 export const App = () => {
+  const [tweetList, setTweetList] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [newTweet, setNewTweet] = useState('');
+
+  const LIKES_URL = (tweetId) => `https://happy-thoughts-technigo.herokuapp.com/thoughts/${tweetId}/like`;
```
[the remark this answers is not in the document]
Comment (petrasson) [reply code/src/App.js:10]: That is a great suggestion, but unfortunately it complains when I add a variable for that URL. But I will try again when I have more time. :)

File: code/src/components/TweetForm.js (hunk 1 above)
Comment (petrasson) [reply code/src/components/TweetForm.js:7]: Ah, sloppy me!! Thank you.

File: code/src/components/TweetForm.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+            cols="30" />
+          <p className="counter">{140 - newTweet.length} / 140</p>
+
+        </div>
+
+        <div className="button-wrapper">
+          <button
+            className="button-form"
+            type="submit"
+            disabled={newTweet.length < 5 || newTweet.length > 140}>
```
[the remark this answers is not in the document]
Comment (petrasson) [reply code/src/components/TweetForm.js:24]: great suggestion! Will add that to my todo list. :)

File: code/src/index.css (hunk 4 above)
Comment (petrasson) [reply code/src/index.css:184]: 🙏🏽 Thank you

File: code/src/index.css (hunk 3 above)
Comment (petrasson) [reply code/src/index.css:85]: Ah... thank you

**Pull request closed without merging**